crates/sui/Cargo.toml

```
@@ -57,7 +57,7 @@ sui-cluster-test.workspace = true
 sui-execution = { path = "../../sui-execution" }
 sui-faucet.workspace = true
 sui-swarm-config.workspace = true
-sui-graphql-rpc = {workspace = true }
+sui-graphql-rpc = { workspace = true }
```

Either a revert or this might be nice
```suggestion
sui-graphql-rpc.workspace = true
```